gradle/libs.versions.toml

```diff
@@ -110,6 +111,9 @@ libsu-nio = { group = "com.github.topjohnwu.libsu", name = "nio", version.ref =
 # Scrollbars
 scrollbars = { group = "com.github.GIGAMOLE", name = "ComposeScrollbars", version.ref = "scrollbars" }
 
+# Compose Icons
+compose-icons-fontawesome = { group = "com.github.BenjaminHalko.compose-icons", name = "font-awesome", version.ref = "compose-icons" }
```

Comment:
What changes are there compared to the upstream version of the library?

Reply:
My version uses font awesome 6. The biggest change is that it uses XTwitter instead of the Twitter icon.

Reply:
Can you add that as a comment in the toml file?